PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue

```diff
@@ -1,60 +1,57 @@
 <template>
-	<v-container fluid>
-	<v-row align="center" justify="left">
-		<!-- Last name -->
-		<v-col class="d-flex" cols="4" sm="4">
+	<v-container>
+    <v-row align="center" justify="start">
+
```

Comment:
Why is there an extra space here? We should probably for consistency and readability.

Reply:
Removed extra spaces throughout (same change applied to all 3 files)